plugin/pkg/auth/authenticator/password/keystone/keystone.go

```diff
@@ -45,18 +45,18 @@ func (keystoneAuthenticator *KeystoneAuthenticator) AuthenticatePassword(usernam
 		glog.Info("Failed: Starting openstack authenticate client")
 		return nil, false, errors.New("Failed to authenticate")
 	}
-
+//FIXME user_id, not user_name...
 	return &user.DefaultInfo{Name: username}, true, nil
 }
 
 // NewKeystoneAuthenticator returns a password authenticator that validates credentials using openstack keystone
-func NewKeystoneAuthenticator(authURL string) (*KeystoneAuthenticator, error) {
-	if !strings.HasPrefix(authURL, "https") {
-		return nil, errors.New("Auth URL should be secure and start with https")
-	}
-	if authURL == "" {
-		return nil, errors.New("Auth URL is empty")
+func NewKeystoneAuthenticator(configPath string) (*KeystoneAuthenticator, error) {
+	if configPath != "" {
+		cfg, _, err := keystonetoken.LoadConfig(configPath)
```

Comment:
I feel there will be problems, I mean I could not have password authn only due to the reference to token authn. maybe we could abstract out this line of code, I mean the `LoadConfig` call..